packages/ui/src/views/connectors/connectors-list/connectors-list.tsx

```diff
@@ -13,13 +26,34 @@ const Title = ({ title }: { title: string }): JSX.Element => (
   </span>
 )
 
-const ConnectivityStatus = ({ item }: { item: ConnectorListItem }): JSX.Element => {
+const ConnectivityStatus = ({ item, t }: { item: ConnectorListItem; t: TFunction }): JSX.Element => {
   const isSuccess = item?.status?.toLowerCase() === ExecutionState.SUCCESS.toLowerCase()
   return (
-    <div className="inline-flex items-center gap-2">
-      <Icon name="dot" size={8} className={cn(isSuccess ? 'text-icons-success' : 'text-icons-danger')} />
-      <Text className="text-cn-foreground-2">{isSuccess ? 'Success' : 'Failed'}</Text>
-    </div>
+    <HoverCard.Root>
+      <HoverCard.Trigger asChild>
+        <Button className="group h-auto gap-2 p-0 font-normal hover:bg-transparent" variant="ghost">
+          <Icon name="dot" size={8} className={cn(isSuccess ? 'text-icons-success' : 'text-icons-danger')} />
+          <Text className="group-hover:text-cn-foreground-1 transition-colors duration-200" color="secondary">
+            {isSuccess
+              ? t('views:connectors.status.success', 'Success')
+              : t('views:connectors.status.failure', 'Failed')}
+          </Text>
+        </Button>
+      </HoverCard.Trigger>
+      <HoverCard.Content className="w-72 whitespace-normal">
+        {/* TODO: need to provide real data */}
```

Comment:
we cannot commit hardcoded sample data.
let's remove this, or hook it up via optional props (to keep it backwards compatible).

Reply:
+1, the error and any related data should come via prop to this component